src/resolver.rs

```diff
@@ -157,7 +157,7 @@ impl ResolverMap {
     }
 
     pub fn new(
-        sym_srcs: &[SymbolSrcCfg],
+        sym_srcs: &[&SymbolSrcCfg],
```

Comment:
do we still need a slice of sources here?

Reply:
Right now yes, because that's what the implementation assumes. But I will work on simplifying that next (the entire context of `ResolverMap` should go).